modules/ROOT/pages/users.adoc

```diff
@@ -60,7 +62,25 @@ By default, in every new organization and business group, you get three options:
 Roles and permissions are grouped under organizations (and also, optionally, under business groups). This means that you can only assign roles and permissions that are related to resources that exist in the organization and/or business group that you are selecting.
 --
 
-If necessary, you can also remove user permissions, but keep in mind that if users have been granted their permission through an assigned role, you cannot view or remove those role-based permissions here--you must do it using the xref:roles.adoc[Roles] section.
+If necessary, you can also remove user permissions, but note that if users have been granted their permissions through an assigned role, you can remove role-based permissions using only the xref:roles.adoc[Roles] functionality. You cannot view or remove role-based permissions in the *Users* page.
```

Comment:
```suggestion
If necessary, you can also remove user permissions, but note that if those permissions are granted through an assigned role, you can remove them only by using the xref:roles.adoc[Roles] functionality. You cannot view or remove role-based permissions in the *Users* page.
```
But...CAN you remove them using the Roles functionality, if that functionality is deprecated?

Reply:
For now, yes.